### ISSUE TITLE
Feature request: Symmetry tool preview

### ISSUE THREAD

**Issue by StainyGrimes** [nvrmind/IssueTracker#15]
When mirroring and flipping it would be very helpful to see where exactly the mirrored/flipped strokes are going to land. Pull trigger to preview, let go to commit?

**Comment by daseyb** [comment]
Yes, that sounds like a good addition and doesn't even require any additional UI. I'll see why I can do there!
